src/sentry/static/sentry/app/components/events/eventDataSection.jsx

```diff
@@ -10,11 +10,13 @@ class EventDataSection extends React.Component {
     wrapTitle: PropTypes.bool,
     toggleRaw: PropTypes.func,
     raw: PropTypes.bool,
+    showGuides: PropTypes.bool,
   };
 
   static defaultProps = {
     wrapTitle: true,
     raw: false,
+    showGuides: true,
```

Comment:
I'd be okay with `hideGuide` too, since then the default would be false and when you add the prop it can just be `<Component hideGuide />` instead of `showGuide={false}`